src/app/pages/charging-stations/charging-station/parameters/charging-station-parameters.component.ts

```diff
@@ -164,6 +170,13 @@ export class ChargingStationParametersComponent implements OnInit, OnChanges {
       if (this.forceInactive) {
         this.forceInactive.setValue(this.chargingStation.forceInactive);
       }
+      if (this.manualConfiguration) {
+        this.manualConfiguration.setValue(this.chargingStation.manualConfiguration);
+      }
+      // If no charge points are available, charging station is manual configurable
+      if (!(this.chargingStation.chargePoints?.length > 0)) {
```

Comment:
It's the same issue regarding the data structure as the one backend side: `chargePoints` must type `ChargePoint[]` as it's defined in the ChargingStation type. You have to reuse array handling helpers and send to the backend `ChargePoint` array. And the backend must expect that date type.